pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -443,18 +446,18 @@ def newpkgcb(pkgId, name, arch):
                         url=modules_url
                     )
                     default_content = ModulemdDefaults(**default)
-                    modulemd_defaults_pb.increment()
```

Comment:
Same issue here where I had changed it to set ```done``` inititally and somehow missed the ```increment()```